lib/lib_rss.php

```diff
@@ -189,7 +189,7 @@ function customSimplePie($attributes = array()) {
 	}
 	if (!empty($attributes['curl_params']) && is_array($attributes['curl_params'])) {
 		foreach ($attributes['curl_params'] as $co => $v) {
-			curl_setopt($ch, $co, $v);
+			$curl_options[$co] = $v;
```

Comment:
As a future improvement, could you please check whether a version without loop and using e.g. `array_merge()` could do the job?

Reply:
I already did, it should work.

Reply:
We need to double check that the array keys are not modified in a wrong way (I am afraid they would be, with `array_merge`)